internal/command/deploy/plan.go

```diff
@@ -22,6 +21,36 @@ import (
 	"golang.org/x/sync/errgroup"
 )
 
+const (
+	stoppedMachinesPoolSize = 30
+)
+
+type MachineLogger struct {
+	store map[string]statuslogger.StatusLine
```

Comment:
Shouldn't we have a mutex since there is are lot concurrent code around this `store` map?

Reply:
all actions on `store` are reads, which is concurrent safe. plus, locking could also mess up how the statuses are displayed.